projects/ion/src/lib/core/types/select.ts

```diff
@@ -8,6 +8,7 @@ export interface IonSelectProps {
   placeholder?: string;
   options?: DropdownItem[];
   events?: EventEmitter<DropdownItem[]>;
+  maxSelecteds?: number;
```

Comment:
```suggestion
  maxSelected?: number;
```
I think that is better this way, what do you think?